src/flexmeasures_client/client.py

```diff
@@ -163,6 +165,11 @@ async def request_once(
             json=json,
             ssl=self.ssl,
         )
+        payload = await response.json()
```

Comment:
To be consistent with the format above, I would format the logs, e.g. `"response status: {response.status}`

Reply:
Good catch, updated